set.mm

```diff
@@ -544674,14 +544683,6 @@ the next (since the empty set has a finite subcover, the
       IJEKOBLZGQDGZCQORODQADMIJFKN $.
   $}
 
-  ${
-    selconj.1 $e |- ( ph <-> ( ps /\ ch ) ) $.
-    $( An inference for selecting one of a list of conjuncts.  (Contributed by
-       Giovanni Mascellani, 23-May-2019.) $)
-    selconj $p |- ( ( et /\ ph ) <-> ( ps /\ ( et /\ ch ) ) ) $=
-      ( wa anbi2i an12 bitr4i ) DAFDBCFZFBDCFFAJDEGBDCHI $.
-  $}
-
```

Comment:
A theorem must not be (re)moved from a mathbox to another mathbox, at least not without permission of the owner. The genereal rule is: if a theorem in a mathbox is used within another mathbox, the theorem should be moved to main! See item 5. in "Mathbox guidelines".

Reply:
After looking at its statement, I would actually "inline it" in the proofs using it, since it is too specific.  There is the command `MM-PA> EXPAND selconj` (followed by `MM-PA> MINIMIZE_WITH *`) for that.

Reply:
@avekens of course I mentioned @giomasce in my commit. I chose this temporary option because I did not want to put it directly into Main: @benjub is right, this is a too specific theorem. Now I put it back to its original place and solved the proof without it.